include/seqan3/contrib/parallel/buffer_queue.hpp

```diff
@@ -141,7 +141,7 @@ class buffer_queue
     {
         spin_delay delay{};
 
-        value_type value;
+        value_type value{};
```

Comment:
This was actually a "may be used uninitialised"... maybe that was our problem on FreeBSD, too?

Reply:
unlikely, but let's see